proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -69,6 +69,14 @@ message RegistryConfig {
     GoConfig go_config = 1;
     NPMConfig npm_config = 2;
   }
+  // Reserved for future remote registry types.
+  reserved 3 to 9;
```

Comment:
Felt nice to group together the different registry config fields (since the oneof above will grow over time) but I can remove this and just set the options field number to 3 if preferred.